Dockerfile.dapper

```diff
@@ -62,19 +59,59 @@ RUN mkdir /usr/tmp && \
     mv /usr/tmp/helm /usr/bin/helm_v3 && \
     chmod +x /usr/bin/kustomize
 
-RUN curl -sLf ${!K3S_DOWNLOAD_URL} -o /usr/bin/k3s && \
-    chmod +x /usr/bin/k3s && \
-    mkdir -p /go/src/github.com/rancher/rancher/.kube && \
+# Set up K3s: copy the necessary binaries from an archive with extracted version of K3s.
+COPY --from=rancher/k3s:v1.22.3-k3s1 \
+    /bin/blkid \
+    /bin/charon \
+    /bin/check-config \
```

Comment:
you can drop this; it's only needed for the `k3s check-config` command which won't be used inside the rancher image